code/modules/mob/living/simple_animal/hostile/headcrab.dm

```diff
@@ -104,3 +104,69 @@
 		else if(host_species == "Gray")
 			I = image('icons/mob/headcrab.dmi', icon_state = "headcrabpod_gray")
 		overlays += I
+
+
+
+/mob/living/simple_animal/hostile/headcrab/fast
+	name = "fast headcrab"
+	desc = "A fast parasitic creature that would like to connect with your brain stem."
+	icon = 'icons/mob/headcrab.dmi'
+	icon_state = "fast_headcrab"
+	icon_living = "fast_headcrab"
+	icon_dead = "fast_headcrab_dead"
+	health = 30
+	maxHealth = 30
+	ranged_cooldown_time = 30
+	jumpdistance = 8
+	jumpspeed = 2
+	speak_emote = list("screech")
+
+/mob/living/simple_animal/hostile/headcrab/fast/update_icons()
+	..()
```

Comment:
. = ..() please so it return its parent call

Reply:
should be implemented now